### ISSUE TITLE
Bug Report: Editor Freezes on Startup Level Opening

### ISSUE THREAD

**Issue by ParasiteSt** [o3de/o3de#10685]
**Describe the bug**
This issue has been occurring for a few weeks now, and it seems to only happen once in a while. The reason I think it is happening is because everything when opening a level tries to load in all at once, and this freezes the editor if there is enough entities with meshes in the scene.

**Steps to reproduce**
The steps are unclear because it is not reproducible, it just happens every once in a while when opening the editor and opening up a level.

**Expected behavior**
The levels would open as expected with no freeze when trying to load all the entities.

**Actual behavior**
The editor freezes up with no response (waited about 10 minutes a few times) until closed and reopened, and the issue disappears until a later time opening the editor and level.

**Found in Branch**
Development [37ce7d4]

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version: 11

**Additional context**
A solution I thought may stop this issue is loading the level entities not all at once and going down a list for spawning in the editor. Just a thought, not sure how or why this issue has been occurring.

**Comment by monroegm** [comment]
Need to figure out some way to get a better repro for this. Are you able to share a project?

**Comment by ParasiteSt** [comment]
I am going to close this issue as I believe I have found what was causing it. I now launch the AP separate from the editor, so I have all assets processed before starting the editor, this seemed to solve any crashing issues that use to happen. This issue could have also been caused by something else totally unrelated as it was a very confusing crash, but it has not happened for a couple months now so I am assuming it has been solved from a PR that has went in a while back.
